conduit/Data/Conduit/Internal/Conduit.hs

```diff
@@ -842,6 +842,9 @@ yield :: Monad m
 yield o = yieldOr o (return ())
 {-# INLINE yield #-}
 
+-- | Send a monadic value to the next component to consume.
+--
+-- Since 1.2.6.7
```

Comment:
Could you use the newer `@since` syntax? I know most of the other docs don't do that yet...